sdk/python/feast/field.py

```diff
@@ -46,6 +50,14 @@ def dtype(self) -> ValueType:
         """
         return self._dtype
 
+    @property
+    def labels(self):
```

Comment:
Is there a reason why some code is on the feature and some of it is on the field? Can we label entities? If not, why not?

Reply:
I am following the current discussion that the use case people need are for the feature currently. So I don't want to assume that people need for the entities as well, Do we need metadata for the entity?

Reply:
It's not so much about entity vs feature as it is feature vs field. Entities inherit from Fields. If the Field can have a label then an Entity can have a label. So we should either limit this entirely to features, or we should add labels to both entities and features.

Reply:
Looking at the .proto files on this branch, I can see that labels field is only applied to Features (not Entities or Fields). Not sure how to proceed with this one as I'm not familiar with requirements/use cases of the Python client. Any suggestion?
Also - my team's use case will require labels on Feature Sets as well and I understand I'm allowed to add it there?

Reply:
Actually `Field`s are the carrier of (feature and entity) values—actual data, not the metadata of registry specifications. I think these changes to `field.py` should probably be backed out.

From discussion Friday, it sounds like we're in agreement to leave entity labeling out of scope for now. Useful thing to have someday (perhaps when #405 gets focus), but it wasn't initially planned for Feast 0.5. So I think we should proceed with adding `map<string, string> labels` to `FeatureSetSpec` as we discussed and providing the API for that.

We might have to defer Python SDK at least in part to the Gojek team, we don't currently use it. It's the primary means of verifying end-to-end functional integration in the project, but maybe as a compromise we can demonstrate the implementation using `grpc_cli` examples as a test plan, and Python SDK could be finished in a second PR.

@woop Does this sound reasonable?

Reply:
Btw @suwik have a look through `tests/e2e` directory in the root of the project for examples of Python SDK, useful for learning the system. They're kicked off for CI by scripts in `infra/scripts` (used to be in `.prow/scripts` at the point of our internal fork).

Reply:
Thanks for suggestion @ches. I'm thinking it would be good to add some logic testing the labels to the e2e tests (and later we could take that to our repo too). I tried to follow the installation instructions in the readme, but after running `docker-compose up` none of the mentioned ports are open. Any suggestion how to get my environment running for this repo so that I'm able to run the e2e tests?

Reply:
I have removed the python SDK changes related to labels (implemented inside field.py). I did that in a separate commit so it should be easy to rollback in case we actually need them.